tensorflow/contrib/eager/python/examples/nmt_with_attention/nmt_with_attention.ipynb

```diff
@@ -677,21 +678,23 @@
         "                # using teacher forcing\n",
         "                dec_input = tf.expand_dims(targ[:, t], 1)\n",
         "        \n",
-        "        total_loss += (loss / int(targ.shape[1]))\n",
-        "        \n",
         "        variables = encoder.variables + decoder.variables\n",
         "        \n",
         "        gradients = tape.gradient(loss, variables)\n",
-        "      \n",
+        "        \n",
         "        optimizer.apply_gradients(zip(gradients, variables), tf.train.get_or_create_global_step())\n",
-        "\n",
+        "        \n",
+        "        batch_loss += (loss / int(targ.shape[1]))\n",
+        "        \n",
+        "        total_loss += batch_loss\n",
+        "        \n",
```

Comment:
I don't think we need this because the batch loss printing that's happening indicates the loss of that particular batch.

Reply:
Ah it is the same functionality as before. I just assigned `loss/int(targ.shape[1]` to this variable `batch_loss` since it's being used in two places - once in `total_loss += loss/int(targ.shape[1]` and the other time during printing below. I meant to assign as `batch_loss = (loss / int(targ.shape[1]))` instead of doing `batch_loss += (loss / int(targ.shape[1]))`. So fixed that.

Reply:
Ah it is the same functionality as before. I just assigned loss/int(targ.shape[1] to this variable batch_loss since it's being used in two places - once in total_loss += loss/int(targ.shape[1] and the other time during printing below. I meant to assign as batch_loss = (loss / int(targ.shape[1])) instead of doing batch_loss += (loss / int(targ.shape[1])). So fixed that.